### PR TITLE
RUM proxy configuration: add intake url for previous sdk version

### PR DESCRIPTION
**Pull request by bcaudan**
### What does this PR do?

List intake url patterns for sdk version < v4

### Motivation

Follow up of #15074 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by liywjl — APPROVED** [review]

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 1]
```diff
@@ -74,16 +74,29 @@ To successfully proxy request to Datadog:
 2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
 4. The request body must remain unchanged.
 
-**IMPORTANT**: Ensure the `ddforward` attribute points to a valid RUM endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:
+**IMPORTANT**: Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:
 
-| Site    | Valid RUM URL Pattern                          | Site Parameter (SDK [initialization parameter][3])| 
+For latest SDK version:
```
Comment (BenoitZugmeyer) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:79]: **🥜 nitpick:** ‏Maybe use tabs to have a more succinct output ([syntax example](https://github.com/DataDog/browser-sdk/pull/1648/commits/1591aecb5e70abe72c7dfe01000258a42049dc6e#diff-a9c53af1f3e9e5b103b5bdef39b52eac991688984575a03139b6fcfdf143ecc5R40))

**Review by BenoitZugmeyer — APPROVED** [review]

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 2]
```diff
@@ -74,16 +74,34 @@ To successfully proxy request to Datadog:
 2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
 4. The request body must remain unchanged.
 
-**IMPORTANT**: Ensure the `ddforward` attribute points to a valid RUM endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:
+**IMPORTANT**: Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:77]:
```suggestion
Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failure to do so may expose your systems to SSRF/XSS attacks. 

The site parameter is an SDK [initialization parameter][3]. Valid intake URL patterns for each site are listed below:
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 3]
```diff
@@ -74,16 +74,34 @@ To successfully proxy request to Datadog:
...
 
-| Site    | Valid RUM URL Pattern                          | Site Parameter (SDK [initialization parameter][3])| 
+{{< tabs >}}
+{{% tab "latest version" %}}
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:80]:
```suggestion
{{% tab "Latest version" %}}
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 4]
```diff
@@ -74,16 +74,34 @@ To successfully proxy request to Datadog:
...
+| Site    | Valid intake URL Pattern                       | Site Parameter      |
 |---------|------------------------------------------------|---------------------|
 | US1     | `https://*.browser-intake-datadoghq.com/*`     | `datadoghq.com`     |
 | US3     | `https://*.browser-intake-us3-datadoghq.com/*` | `us3.datadoghq.com` |
 | US5     | `https://*.browser-intake-us5-datadoghq.com/*` | `us5.datadoghq.com` |
 | EU1     | `https://*.browser-intake-datadoghq.eu/*`      | `datadoghq.eu`      |
 | US1-FED | `https://*.browser-intake-ddog-gov.com/*`      | `ddog-gov.com`      |
 
+{{% /tab %}}
+{{% tab "before v4" %}}
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:91]:
```suggestion
{{% tab "Before `v4`" %}}
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 5]
```diff
@@ -74,16 +74,34 @@ To successfully proxy request to Datadog:
...
+|---------|--------------------------------------|---------------------|
+| US1     | `https://*.logs.datadoghq.com/*`     | `datadoghq.com`     |
+| US3     | `https://*.logs.us3-datadoghq.com/*` | `us3.datadoghq.com` |
+| EU1     | `https://*.logs.datadoghq.eu/*`      | `datadoghq.eu`      |
+| US1-FED | `https://*.logs.ddog-gov.com/*`      | `ddog-gov.com`      |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+**Note:** Site parameter is an SDK [initialization parameter][3].
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:103]:
```suggestion
```

**Review by alai97 — CHANGES_REQUESTED** [review]

**Review by alai97 — APPROVED** [review]

**Pull request merged**